cmd/user-agent/src/pages/GetVC.vue

```diff
@@ -276,10 +417,59 @@ SPDX-License-Identifier: Apache-2.0
                 // Call Credential Handler callback
                 this.credentialEvent.respondWith(new Promise(function (resolve) {
                     return resolve({
-                        dataType: "Response",
+                        dataType: "VerifiablePresentation",
                         data: data
                     });
                 }))
+            },
+            createPresentation: async function () {
+                let creds = await this.getSelectedCredentials()
+                if (creds.retry) {
+                    this.errors.push(creds.retry)
+                    return
+                }
+
+                let data
+                if (creds.vcs) {
+                    await this.aries.verifiable.generatePresentation({
+                        verifiableCredential: creds.vcs,
+                        did: this.issuers[this.selectedIssuer].key,
+                        domain: this.domain,
+                        challenge: this.challenge,
+                        // TODO can be an option in view
+                        skipVerify: true,
+                        signatureType:"JsonWebSignature2020",
```

Comment:
why is this hardcoded?

Reply:
@fqutishat ^^
It is been added in previous commit, aries doesn't choose defaul anymoret, how can we make it as an option in CHAPI window?
@troyronda

Reply:
@sudeshrshetty no need

Reply:
I'm fixing that